WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -353,7 +353,7 @@ public void Initialize(NodesCollection nodes, WasabiSynchronizer synchronizer)
 						catch (Exception ex)
 						{
 							Logger.LogError(ex);
-							NotificationHelpers.Error($"Could not get Legal Documents!{(ex is ConnectionException ? " Backend not connected." : "")}");
+							NotificationHelpers.Error($"Could not get Legal Documents! Check your Internet connection!{(ex is ConnectionException ? " Backend not connected." : "")}");
```

Comment:
If the exception is not related the connection the user would still get `Check your Internet connection!` which is wrong.

```suggestion
							NotificationHelpers.Error($"Could not get Legal Documents!{(ex is ConnectionException ? " Backend not connected. Check your internet connection!" : "")}");
```